nrf_modem/doc/ug_nrf_modem_porting_os.rst

```diff
@@ -263,6 +263,31 @@ nrf_modem_os_sem_count_get()
 
 This function is called to retrieve the count of a semaphore.
 
+nrf_modem_os_mutex_init()
+-----------------------
+
+This function is called by the library to allocate and initialize a mutex.
+
+*Required action*:
+
+* Allocate and initialize a mutex.
+* If an address of an already allocated mutex is provided as an input, the allocation part is skipped and the mutex is only reinitialized.
+
+.. note::
+
+   Mutexes are not required if multithreaded access to modem functionality is not needed.
+   In this case, the function can blindly return ``0``.
```

Comment:
Not sure if this line is necessary, It is customary to return 0 or -1 on non-implemented functions. If we want to enforce a specific value then we must say so.

Reply:
Reworded to use `must`.